spec/factories/facility.rb

```diff
@@ -1,8 +1,7 @@
 FactoryBot.define do
   factory :facility do
-    val = Faker::Name.unique.name
-    name { val }
-    code { val.gsub(/[aeiou|AEIOU|\s]+/, '').strip }
-    organization_id { FactoryBot.create(:organization).id }
+    name { Faker::Games::HalfLife.location.to_s }
```

Comment:
Making a note for myself to clean up usage of `Faker`. I dont think we really need it.

Reply:
😢  but fair enough